l10n_es_mis_report/data/mis_report_balance_normal.xml

```diff
@@ -0,0 +1,2473 @@
+            <field name="type">num</field>
+            <field name="compare_method">pct</field>
+            <field name="sequence">730</field>
+            <field name="name">es12720</field>
+            <field name="description">2. Otros activos líquidos equivalentes</field>
+            <field name="expression">+bale[576%]</field>
+            <field name="auto_expand_accounts">True</field>
+            <field
+                name="auto_expand_accounts_style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l4i"
+            />
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l4"
+            />
+        </record>
+        <record id="mis_report_kpi_es_balance_normal_10000" model="mis.report.kpi">
+            <field name="report_id" ref="mis_report_es_balance_normal" />
+            <field name="type">num</field>
+            <field name="compare_method">pct</field>
+            <field name="sequence">740</field>
+            <field name="name">es10000</field>
+            <field name="description">TOTAL ACTIVO (A + B)</field>
+            <field name="expression">+es11000+es12000</field>
+            <field name="auto_expand_accounts">True</field>
+            <field
+                name="auto_expand_accounts_style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l1i"
+            />
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l1"
+            />
+        </record>
+        <record id="mis_report_kpi_es_balance_normal_20000" model="mis.report.kpi">
+            <field name="report_id" ref="mis_report_es_balance_normal" />
+            <field name="type">num</field>
+            <field name="compare_method">pct</field>
+            <field name="sequence">750</field>
+            <field name="name">es20000</field>
+            <field name="description">A) PATRIMONIO NETO</field>
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l1"
+            />
+            <field name="expression">+es21000 +es22000 +es23000</field>
+        </record>
+        <record id="mis_report_kpi_es_balance_normal_21000" model="mis.report.kpi">
+            <field name="report_id" ref="mis_report_es_balance_normal" />
+            <field name="type">num</field>
+            <field name="compare_method">pct</field>
+            <field name="sequence">760</field>
+            <field name="name">es21000</field>
+            <field name="description">A-1) Fondos propios</field>
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l2"
+            />
+            <field name="expression">
+                +es21100 +es21200 +es21300 +es21400 +es21500 +es21600 +es21700 +es21800
+                +es21900
```

Comment:
Please, review all of lines because some generate in new lines and spaces and the balance shows error in the result.
In odoo I saw this result of this line
+es21100 +es21200 +es21300 +es21400 +es21500 +es21600 +es21700 +es21800                +es21900
There are some spaces betwenn +es21800 and +es21900